sqlest/src/main/scala/sqlest/executor/Database.scala

```diff
@@ -300,53 +306,60 @@ case class Transaction(database: Database) extends Session(database) {
     withConnection { connection =>
       val (preprocessedCommand, sql, argumentLists) = database.statementBuilder(command)
       val startTime = new DateTime
-      val preparedStatement = prepareStatement(connection, preprocessedCommand, sql, argumentLists)
+
       try {
```

Comment:
Is this for when we creating the preparedStatement throws we capture the exception?